frontend/packages/helm-plugin/src/catalog/components/HelmReadmeLoader.tsx

```diff
@@ -19,7 +25,9 @@ const HelmReadmeLoader: React.FC<HelmReadmeLoaderProps> = ({ chartURL }) => {
       let chartData;
 
       try {
-        chartData = await coFetchJSON(`/api/helm/chart?url=${chartURL}`);
+        chartData = await coFetchJSON(
+          `/api/helm/chart?url=${chartURL}&namespace=${namespace}&indexEntry=${chartIndexEntry}`,
+        );
```

Comment:
We need to escape at least `chartURL` here, right?

Namespace names are "safe" for any injection because of their limited character subset, but we might escape all parameters.